cmake/SetupCompilerOptions.cmake

```diff
@@ -323,13 +323,15 @@ blt_append_custom_compiler_flag(
 #
 
 if ( COMPILER_FAMILY_IS_MSVC AND NOT BUILD_SHARED_LIBS )
-    foreach(flag_var
-            CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE
-            CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO)
-        if(${flag_var} MATCHES "/MD")
-            string(REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
-        endif(${flag_var} MATCHES "/MD")
-    endforeach(flag_var)
+  foreach(_lang C CXX)
+    foreach(_flag
```

Comment:
@davidbeckingsale -- Since we have to take another pass to pass the CI, would you mind changing it to the following?

```cmake
foreach(_lang C CXX)
    foreach(_build
            FLAGS FLAGS_DEBUG FLAGS_RELEASE
            FLAGS_MINSIZEREL FLAGS_RELWITHDEBINFO) 
        set(_flag CMAKE_${_lang}_${_build})
        if(${_flag} MATCHES "/MD")
            string(REGEX REPLACE "/MD" "/MT" ${_flag} "${${_flag}}")
        endif()
    endforeach()
endforeach()
```

Reply:
Of course, pushed 👍

Reply:
Thanks